backend/src/api/pe-reports.ts

```diff
@@ -0,0 +1,45 @@
+import { wrapHandler } from './helpers';
+import S3Client from '../tasks/s3-client';
+
+export const export_pe_report = wrapHandler(async (event) => {
+  const Key = JSON.parse(event.body!).Key;
+  const client = new S3Client();
+  const url = await client.exportPeReport(Key);
+  if (url == 'File does not exist') {
+    return {
+      statusCode: 404,
+      body: ''
+    };
+  }
+  return {
+    statusCode: 200,
+    body: JSON.stringify({ url })
+  };
+});
+
+export const upload_pe_report = wrapHandler(async (event) => {
```

Comment:
For each endpoint, let's add a swagger definition by adding a comment as done here: https://github.com/cisagov/crossfeed/blob/master/backend/src/api/domains.ts#L159-L167